alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,66 @@
+
+## Impact
+Errors on the aggregated API can result in the unavailability of some openshift services.  
+
+## Diagnosis
+
+##### Check the APIs status checks are on True
+
+Currently there are 3 aggregated APIs in an Openshift Cluster. The API on the `openshift-apiserver` namespace, the prometheus-adapter on the namespace `openshift-monitoring`, and the packageserver-service in the `openshift-operator-lifecycle-manager` namespace.
+
+To get a list of the aggregated APIs and the apiGroup to whom they answer requests use the following command:
+
+```shell
+$ oc get apiservice -n openshift-apiserver | grep openshift-
+v1.apps.openshift.io                            openshift-apiserver/api                                      True        340d
+v1.authorization.openshift.io                   openshift-apiserver/api                                      True        340d
+v1.build.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.image.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.oauth.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.packages.operators.coreos.com                openshift-operator-lifecycle-manager/packageserver-service   True        340d
+v1.project.openshift.io                         openshift-apiserver/api                                      True        340d
+v1.quota.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.route.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.security.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.template.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.user.openshift.io                            openshift-apiserver/api                                      True        340d
+v1beta1.metrics.k8s.io                          openshift-monitoring/prometheus-adapter                      True        340d
+```
+Here we want the availability status to be true. If there is a `False` somewhere that means that requests made to that APIservice, APIserver pods or resources belonging to that apiGroup failed many times during the last 5 mins.
+
+##### Check the `apiserver` pods
+```shell
+$ oc get pods -n openshift-apiserver
+NAME                        READY   STATUS    RESTARTS   AGE
+apiserver-c74d6bd79-pkmw4   1/1     Running   0          41d
+apiserver-c74d6bd79-t4l9d   1/1     Running   0          41d
+apiserver-c74d6bd79-z6qxx   1/1     Running   0          41d
+```
+If their status is not `Running` check the logs for more details. 
+
+##### Check the authentication certificates of the aggregated API
```

Comment:
Why we need check the certificate, what we can do if the cert expired.

Reply:
Unfortunately, I wasn't able to go into full depth with that. Our current runbook only mentions this issue that short. I would just leave it like this since this can only be seen as the initial version or templates and the runbooks have to be improved as time passes.